superdesk/io/feed_parsers/scoop_newsml_2_0.py

```diff
@@ -0,0 +1,118 @@
+
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+import pytz
+import datetime
+from .newsml_2_0 import NewsMLTwoFeedParser
+from superdesk.io import register_feed_parser
+from superdesk.errors import ParserError
+from superdesk.metadata.item import ITEM_TYPE
+from superdesk.io.iptc import subject_codes
+from superdesk.etree import get_word_count
+
+class ScoopNewsMLTwoFeedParser(NewsMLTwoFeedParser):
+
+    NAME = 'scoop_newsml2'
+
+    def can_parse(self, xml):
+        if xml.tag.endswith('newsMessage'):
+            try:
+                self.root = xml
+                header = xml.find(self.qname('Header'))
+                origin = header.find(self.qname('origin')).text
+                if origin == 'BusinessDesk':
```

Comment:
The parser is specifically for this source, I doubt any other provider would implement the same "quirks" in the standards.